### PR TITLE
Automated code formatting

### PR DESCRIPTION
**Pull request by knservis**
The actual changes are all in the first commit. All the rest is whitespace.

### CODE REVIEW

File: 3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/EtlPipeline.scala [hunk 1]
```diff
@@ -35,14 +32,8 @@ import scalaz._
...
 // This project
-import adapters.{
-  RawEvent,
-  AdapterRegistry
-}
-import enrichments.{
-  EnrichmentRegistry,
-  EnrichmentManager
-}
+import adapters.{RawEvent,              AdapterRegistry}
```
Comment (chuwy) [inline 3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/EtlPipeline.scala]: This looks quite wrong for me.

File: 3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/adapters/AdapterRegistry.scala [hunk 2]
```diff
@@ -25,8 +25,8 @@ import Scalaz._
 
 // This project
 import loaders.CollectorPayload
-import registry.snowplow.{Tp1Adapter => SpTp1Adapter}
-import registry.snowplow.{Tp2Adapter => SpTp2Adapter}
+import registry.snowplow.{Tp1Adapter      => SpTp1Adapter}
+import registry.snowplow.{Tp2Adapter      => SpTp2Adapter}
```
Comment (chuwy) [inline 3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/adapters/AdapterRegistry.scala]: Doesn't seem useful.

File: 3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/adapters/registry/Adapter.scala [hunk 3]
```diff
@@ -96,7 +96,7 @@ trait Adapter {
 
     val j1 = json transformField {
       case (k, v) => {
-        if (k == tsFieldKey) {
+        if(k == tsFieldKey) {
```
Comment (chuwy) [inline 3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/adapters/registry/Adapter.scala]: Very confusing.

File: 3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/adapters/registry/CallrailAdapter.scala [hunk 4]
```diff
@@ -63,8 +60,8 @@ object CallrailAdapter extends Adapter {
 
   // Create a simple formatter function
   private val CallrailFormatter: FormatterFunc = {
-    val bools = List("first_call", "answered")
-    val ints = List("duration")
+    val bools                        = List("first_call", "answered")
```
Comment (chuwy) [inline 3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/adapters/registry/CallrailAdapter.scala]: I don't really understand this horizontal alignment.

**Comment by alexanderdean** [comment]
Better commit messages:

```
Scala Common Enrich: configure automated code formatting (closes #3496)
Scala Common Enrich: apply automated code formatting (closes #xxxx)
```

**Comment by knservis** [comment]
@chuwy It is not their fault. I have added some of the more exaggerated configurations (look at .scalafmt.conf align.tokens ). Overall we can play with other configurations and if they we need to switch them off somewhere as an exception we can always go ```//format: off```.  Have a look here for configurations: http://scalameta.org/scalafmt/#Configuration

File: 3-enrich/scala-common-enrich/.scalafmt.conf [hunk 5]
```diff
@@ -0,0 +1,12 @@
+align = most
+maxColumn = 140
+docstrings = JavaDoc
+optIn.breakChainOnFirstMethodDot = false
+spaces.afterKeywordBeforeParen = false
```
Comment (BenFradet) [inline 3-enrich/scala-common-enrich/.scalafmt.conf]: let's go with true

File: 3-enrich/scala-common-enrich/.scalafmt.conf [hunk 6]
```diff
@@ -0,0 +1,12 @@
+align = most
```
Comment (BenFradet) [inline 3-enrich/scala-common-enrich/.scalafmt.conf]: let's go with some, that'll remove those weird alignments @chuwy was talking about

Comment (BenFradet) [reply 3-enrich/scala-common-enrich/.scalafmt.conf]: also can we have `style = default` at the top?

File: 3-enrich/scala-common-enrich/.scalafmt.conf [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+maxColumn = 140
+docstrings = JavaDoc
+optIn.breakChainOnFirstMethodDot = false
+spaces.afterKeywordBeforeParen = false
+continuationIndent.defnSite = 2
+rewrite.rules = [
+  RedundantBraces,
+  RedundantParens,
+  PreferCurlyFors
```
Comment (BenFradet) [inline 3-enrich/scala-common-enrich/.scalafmt.conf]: could we add `AsciiSortImports`?

**Pull request merged**